### DISCUSSION TITLE
"Loading more" on infinite scroll TableVirtuoso

### DISCUSSION THREAD

**Discussion by Rednegniw** [petyosi/react-virtuoso#900] [Q&A]
Hi! I am using a `TableVirtuoso` and I also have implemented an infinite scrolling mechanism based on `endReached`. The thing is, I would like to append a `Footer` component from your example to show loading state when loading further items. TableVirtuoso doesn't have it though. Is there another way I'm not seeing?

**Comment by gmoothart** [comment]
+1 I am trying to do the same thing.

**Comment by vishalsolanki-anagram** [comment]
> Hi! I am using a `TableVirtuoso` and I also have implemented an infinite scrolling mechanism based on `endReached`. The thing is, I would like to append a `Footer` component from your example to show loading state when loading further items. TableVirtuoso doesn't have it though. Is there another way I'm not seeing?

i think they are supporting now

```
 components={{ fixedFooterContent={()=>
                footer()
              }
              fixedHeaderContent={() =>
              }}} 
```

              
              these are the two props which u can pass in the component prop to show header and fotter for the table
              
 ```
             function footer (){
  return (
   <>{showLoading && <TableRow>
      <TableCell colSpan={9} align="center">
        <CircularProgress sx={{
          color:'rgba(200, 80, 192, 1)',
        }} size={40}/>
      </TableCell>
    </TableRow>}</>
  )
}
```
 i am using version "^4.13.0"
